lib/semantics/symbol.h

```diff
@@ -691,5 +692,16 @@ inline bool ProcEntityDetails::HasExplicitInterface() const {
   }
   return false;
 }
+
+// SymbolSet
+struct SymbolIsLess {
+  constexpr bool operator()(const std::reference_wrapper<const Symbol> &x,
```

Comment:
Just like Tim, the ```constexpr``` on this line caused compilation problems for me running gcc 8.3.0 on my laptop.  Removing it made everything build and test correctly.

Reply:
Can you please confirm whether the change that you wanted has been made, and clear the block for the merge if so?

Reply:
Oops.  Sorry for the delay.  All looks good.